internal/venice-common/src/main/java/com/linkedin/venice/pubsub/ImmutablePubSubMessage.java

```diff
@@ -82,4 +82,9 @@ public boolean isEndOfBootstrap() {
   public PubSubMessageHeaders getPubSubMessageHeaders() {
     return pubSubMessageHeaders;
   }
+
+  @Override
+  public String toString() {
+    return "PubSubMessage{topicPartition=" + topicPartition + ", offset=" + offset + ", timestamp=" + timestamp + '}';
```

Comment:
Nit: as PubSubTopicPartition already has format like this: TP(topic: '" + topic.getName() + "', partition: " + partitionNumber + "), maybe we can get rid of 'topicPartition='. Besides, do you think adding payloadSize and boolean isEndOfBootstrap will help debugging?

Reply:
Good catch. Thanks. Updated. 

>Besides, do you think adding payloadSize and boolean isEndOfBootstrap will help debugging?
IMO this is not very helpful.

Reply:
Yeah, and what does isEndOfbootstrap means? I see it always returns false.

Reply:
I think it's used in CC context but I don't know it's significance

Reply:
Added `ImmutableChangeCapturePubSubMessage::toString` which logs `isEoB` as well